helm-charts/common/embedding-usvc/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: embedding-svc
+spec:
+  selector:
+    matchLabels:
+      svc: embedding-svc
```

Comment:
Can we just use {{- include "embedding-usvc.labels" . | nindent 4 }} here instead of adding new serivce labels?

Reply:
Yes, should be fine.